plugin.json

```diff
@@ -15,7 +15,7 @@
         "bundle_path": "webapp/dist/main.js"
     },
     "settings_schema": {
-        "header": "To set up the GitHub plugin, [register a GitHub OAuth app](https://github.com/settings/applications/new) with `https://your-mattermost-url.com/plugins/github/oauth/complete` as your Authorization callback URL.",
+        "header": "Subscribe to notifications, stay up-to-date with reviews, see the status of pull requests at a glance, and other common GitHub actions - directly from Mattermost with this GitHub plugin. \n \n [Register a GitHub OAuth app](https://github.com/settings/applications/new) with `https://your-mattermost-url.com/plugins/github/oauth/complete` as your Authorization callback URL, then follow [these instructions](https://www.mattermost.com/pl/default-github-plugin#configuration) to complete the setup.",
```

Comment:
Why do we have partial instructions in the header here? Why not just the link since I have to visit the link anyway.

Reply:
Originally, I believe @jasonblais observed some users who had problems with creation of the OAuth app that originated here. (they didn't use the proper Oauth callback URL in the proper fields when creating the app in GitHub).  We added some extra context for the user, so that even if they rushed - the important information (the proper Callback URL) was here in front of them.

Reply:
Removed instructions on Console page